contracts/augur/ZeroXExchange.sol

```diff
@@ -107,336 +107,6 @@ contract ZeroXExchange is IExchange, ReentrancyGuard {
         ));
     }
 
-    /// @dev Adds properties of both FillResults instances.
-    ///      Modifies the first FillResults instance specified.
-    /// @param totalFillResults Fill results instance that will be added onto.
-    /// @param singleFillResults Fill results instance that will be added to totalFillResults.
-    function addFillResults(FillResults memory totalFillResults, FillResults memory singleFillResults)
-        internal
-        pure
-    {
-        totalFillResults.makerAssetFilledAmount = totalFillResults.makerAssetFilledAmount.add(singleFillResults.makerAssetFilledAmount);
-        totalFillResults.takerAssetFilledAmount = totalFillResults.takerAssetFilledAmount.add(singleFillResults.takerAssetFilledAmount);
-        totalFillResults.makerFeePaid = totalFillResults.makerFeePaid.add(singleFillResults.makerFeePaid);
-        totalFillResults.takerFeePaid = totalFillResults.takerFeePaid.add(singleFillResults.takerFeePaid);
-    }
-
-    /// @dev Fills the input order.
-    ///      Returns false if the transaction would otherwise revert.
-    /// @param order Order struct containing order specifications.
-    /// @param takerAssetFillAmount Desired amount of takerAsset to sell.
-    /// @param signature Proof that order has been created by maker.
-    /// @return Amounts filled and fees paid by maker and taker.
-    function fillOrderNoThrow(
```

Comment:
`fillOrderNoThrow` eventually call `fillOrderInternal` - Calling `exchange.fillOrderInternal`  directly from `ZeroXTrade.sol`

Reply:
In reality we won't actually be able to make such changes. This contract is from 0x. We're currently working on a PR that will encorporate their contracts with minimal changes and we won't be able to diverge from their provided interfaces.